<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>select</class>
 <widget class="QWidget" name="select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>351</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Категории</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Man">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Мужская одежда</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>140</y>
     <width>711</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_PP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Один поставщик и производитель</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Otch">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Отчёт со склада</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Futbolki">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>360</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Футболки</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Kat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Все товары (каталог)</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Child">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Одежда для детей</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Ubki">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Юбки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>152</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Тип одежды</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>20</width>
     <height>341</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Fem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Женская одежда</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Galstuki">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Галстуки</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Sport">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Спортивное</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Rubashki">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Рубашки</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Vzr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Одежда для взрослых</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Djinsi">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Джинсы</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="Box_Noski">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Носки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>701</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Выберите тип данных</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>701</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
